### ISSUE TITLE
DB2 stored procedure nested block support

### ISSUE THREAD

**Issue by franciskittu** [flyway/flyway#1691]
I'm running flyway command-line 4.2.0 against DB2 9.7 FIXPACK 7 on WINDOWS 64bit.

I tried to migrate just one SQL file containing a dummy procedure definition with nested BEGIN END block.
```

CREATE PROCEDURE dummy_proc ()
LANGUAGE SQL
BEGIN
    declare var1 TIMESTAMP;
    
    SELECT CURRENT_TIMESTAMP INTO var1 FROM SYSIBM.DUAL;
    
    BEGIN
        declare var2 DATE;
        SELECT CURRENT_DATE INTO var2 FROM SYSIBM.DUAL;
    END;
END; -- here I tried with and without the ";" char and also with "@" and --#SET TERMINATOR @ at the beginning

```
The statement correctly compile by running it on a  DB2 client. So, the result should be the same on flyway.

Instead, the migration failed due to a -104 error. The same applies to every procedure containing nested blocks.

**Comment by axelfontaine** [comment]
The `--#SET TERMINATOR` case should now work correctly with the fix for #1690 

However without that, nesting is indeed not supported yet.

**Comment by franciskittu** [comment]
Ok, thanks for your support.
